packages/frontend-app-api/src/extensions/Core.tsx

```diff
@@ -33,6 +34,9 @@ export const Core = createExtension({
     components: createExtensionInput({
       component: coreExtensionData.component,
     }),
+    translations: createExtensionInput({
+      translation: createTranslationExtension.translationDataRef,
```

Comment:
if we already have `*coreExtension*` as a thing, could we even double down on this and do `fooExtension.create` + `fooExtension.dataRef` as a standard pair? 🤔

Maybe that collides with our naming discussion yesterday where it might get common to want to write `const fooExtension = fooExtension.create` which obv is a collision.

I do wonder if nesting it like this makes it less discoverable in some ways. For example slightly less intuitive perhaps when using editor completion in an import statement, or when trying to type out `translationDataRef` and auto importing.

So ... not entirely sure about this 🤔

Reply:
Yep I've actually been thinking exactly that, only difference that I had upper case in mind 😅. `Extension.create`, `PageExtension.create`, `TranslationExtension.dataRefs.translation`, etc. Naming convention would simply be switched to `<Kind>Extension`. I don't think it's quite as clear though, feels a bit more complex, but if we end up needing additional functions that interact with extensions of certain kinds then I think we should go there straight away, for example `PageExtension.clone()` or `PageExtension.decorate()`.

Reply:
Oh right regarding discoverability, the intention here is really for these extension data refs to not actually be that discoverable, it's kinda the point. The only time they need to be used is when you create extensions that take this specific kind of extension as input, in all other situations it's better to have the ref out of the way and avoid cluttering the exports.